app/dashboard/views.py

```diff
@@ -4483,6 +4550,13 @@ def save_tribe(request,handle):
             tribe.tribe_description = tribe_description
             tribe.save()
 
+        if request.FILES.get('cover_image'):
```

Comment:
need to validate on server to avoid security issues ex: `invalid_file_response(cover_image, supported=['image/jpeg', 'image/jpg'])`